### ISSUE TITLE
does visual actually build

### ISSUE THREAD

**Issue by idella** [vpython/visual#7]
{{{
/usr/include/features.h:168:0: note: this is the location of the previous definition
/usr/include/python2.7/pyconfig.h:1183:0: warning: "_XOPEN_SOURCE" redefined [enabled by default]
/usr/include/features.h:170:0: note: this is the location of the previous definition
In file included from /usr/include/boost-1_49/boost/python/detail/wrap_python.hpp:50:0,
                 from /usr/include/boost-1_49/boost/python/detail/prefix.hpp:13,
                 from /usr/include/boost-1_49/boost/python/numeric.hpp:8,
                 from ../include/util/vector.hpp:10,
                 from ../include/util/extent.hpp:9,
                 from ./core/util/extent.cpp:6:
/usr/include/python2.7/pyconfig.h:1161:0: warning: "_POSIX_C_SOURCE" redefined [enabled by default]
/usr/include/features.h:168:0: note: this is the location of the previous definition
/usr/include/python2.7/pyconfig.h:1183:0: warning: "_XOPEN_SOURCE" redefined [enabled by default]
/usr/include/features.h:170:0: note: this is the location of the previous definition

---

In file included from /usr/include/boost-1_49/boost/thread/pthread/mutex.hpp:14:0,
                 from /usr/include/boost-1_49/boost/thread/mutex.hpp:16,
                 from ../include/util/thread.hpp:9,
                 from ../include/util/atomic_queue.hpp:4,
                 from ./core/util/atomic_queue.cpp:1:
/usr/include/boost-1_49/boost/thread/xtime.hpp:23:5: error: expected identifier before numeric constant
/usr/include/boost-1_49/boost/thread/xtime.hpp:23:5: error: expected '}' before numeric constant
/usr/include/boost-1_49/boost/thread/xtime.hpp:23:5: error: expected unqualified-id before numeric constant
/usr/include/boost-1_49/boost/thread/xtime.hpp:46:14: error: expected type-specifier before 'system_time'
In file included from /usr/include/boost-1_49/boost/thread/pthread/mutex.hpp:14:0,
                 from /usr/include/boost-1_49/boost/thread/mutex.hpp:16,
                 from ../include/util/thread.hpp:9,
                 from ../include/util/atomic_queue.hpp:4,
                 from ./core/util/atomic_queue.cpp:1:
/usr/include/boost-1_49/boost/thread/xtime.hpp: In function 'int xtime_get(xtime_, int)':
/usr/include/boost-1_49/boost/thread/xtime.hpp:73:40: error: 'get_system_time' was not declared in this scope
/usr/include/boost-1_49/boost/thread/xtime.hpp:73:40: note: suggested alternative:
/usr/include/boost-1_49/boost/thread/thread_time.hpp:19:24: note:   'boost::get_system_time'
/usr/include/boost-1_49/boost/thread/xtime.hpp: At global scope:
/usr/include/boost-1_49/boost/thread/xtime.hpp:88:1: error: expected declaration before '}' token
make[1]: *_\* [atomic_queue.lo] Error 1
make[1]: **\* Waiting for unfinished jobs....
}}}

and so forth

https://bugs.gentoo.org/show_bug.cgi?id=460384

I'd serve you up some build logs but this tracker doesn't seem to offer to take them.

In short, the boost headers are confusing the source .cpp files totally.  They barely get out o the blocks and the build falls over.
What on earth is going on here?????
